app/api-requests/claim-service-api.js

```diff
@@ -63,9 +64,69 @@ function getMostRecentReviewDate (previousClaims, latestVetVisitApplication) {
+  }
+  return { isValid: true, content: {} }
+}
+
+function isValidEndemicsDate (previousClaims, dateOfVisit) {
+  const priorFailedReviewClaims = (previousClaims ?? []).filter((previousClaim) => REJECTED === previousClaim.statusId && previousClaim.type === claimType.review && new Date(previousClaim.data.dateOfVisit) < dateOfVisit)
+  const priorSuccessFulReviewClaims = (previousClaims ?? []).filter((previousClaim) => successfulStatuses.includes(previousClaim.statusId) && previousClaim.type === claimType.review && new Date(previousClaim.data.dateOfVisit) < dateOfVisit)
+  const priorEndemicsClaims = (previousClaims ?? []).filter((previousClaim) => statusesFor10MonthCheck.includes(previousClaim.statusId) && previousClaim.type === claimType.endemics && new Date(previousClaim.data.dateOfVisit) < dateOfVisit)
+  const nextEndemicsClaims = (previousClaims ?? []).filter((previousClaim) => statusesFor10MonthCheck.includes(previousClaim.statusId) && previousClaim.type === claimType.endemics && new Date(previousClaim.data.dateOfVisit) > dateOfVisit)
+  const sortedPriorFailedReviewClaims = _.sortBy(priorFailedReviewClaims, [(priorReviewClaim) => { return new Date(priorReviewClaim.data.dateOfVisit) }])
+  const sortedPriorSuccessFulReviewClaims = _.sortBy(priorSuccessFulReviewClaims, [(priorReviewClaim) => { return new Date(priorReviewClaim.data.dateOfVisit) }])
+  const sortedPriorEndemicsClaims = _.sortBy(priorEndemicsClaims, [(priorEndemicsClaim) => { return new Date(priorEndemicsClaim.data.dateOfVisit) }])
+  const sortedNextEndemicsClaims = _.sortBy(nextEndemicsClaims, [(nextEndemicsClaim) => { return new Date(nextEndemicsClaim.data.dateOfVisit) }])
+  const isValidPriorFailedReviewClaimsDifference = sortedPriorFailedReviewClaims.length ? is10MonthsDifference(dateOfVisit, sortedPriorFailedReviewClaims[sortedPriorFailedReviewClaims.length - 1].data.dateOfVisit, 'lessThanTenMonths') : undefined
+  const isValidPriorSuccessFulReviewClaimsDifference = sortedPriorSuccessFulReviewClaims.length ? is10MonthsDifference(dateOfVisit, sortedPriorSuccessFulReviewClaims[sortedPriorSuccessFulReviewClaims.length - 1].data.dateOfVisit, 'lessThanTenMonths') : undefined
+  const isValidPriorEndemicsDifference = sortedPriorEndemicsClaims.length ? is10MonthsDifference(dateOfVisit, sortedPriorEndemicsClaims[sortedPriorEndemicsClaims.length - 1].data.dateOfVisit) : undefined
+  const isValidNextEndemicsDifference = sortedNextEndemicsClaims.length ? is10MonthsDifference(new Date(sortedNextEndemicsClaims[sortedNextEndemicsClaims.length - 1].data.dateOfVisit), dateOfVisit) : undefined
+
+  console.log('%%%%%%%%%%%%%%%', sortedPriorSuccessFulReviewClaims.length, priorSuccessFulReviewClaims)
+  if (sortedPriorFailedReviewClaims.length && !isValidPriorFailedReviewClaimsDifference) {
+    return { isValid: false, content: { url: '', text: 'The Dairy Farm - SBI 123456789 had a failed review claim for beef cattle in the last 10 months.' } }
```

Comment:
This is not static content - we are meant to be displaying the business name, SBI, and type of species that they are reviewing for